app/Http/Controllers/Store/Controller.php

```diff
@@ -34,4 +34,19 @@ protected function userCart()
             return Order::cart(Auth::user());
         }
     }
+
+    protected function hasPendingCheckout()
+    {
+        $cart = $this->userCart();
+        if ($cart) {
+            return $cart->isProcessing();
+        }
+    }
+
+    protected function redirectIfPendingCheckout()
```

Comment:
not used? (probably something along the line "it doesn't work because actually using it looks stupid")